jgt_tools/check_version.py

```diff
@@ -0,0 +1,25 @@
+"""Ensure version bump."""
+import subprocess
+
+
+def _any_py_files_changed(file_names):
+    return any(file_name.endswith(".py") for file_name in file_names)
```

Comment:
⚠️ `II` ⚠️
What if this had an optional plugin architecture (ala `tag_to_url`) so that if a project needed to be more specific / different, it could, but this would be the default.

Reply:
Done, and tested locally by creating a custom file list checker that prints out the files, and adding that via the pyproject plugin section, running `check-version` and ensuring it ran.